modelservice/src/main/resources/config/application.yml

```diff
@@ -27,6 +23,8 @@ management:
 
 server:
   port: 8080
- 
- 
- 
\ No newline at end of file
+  
+db:
+  url: "jdbc:mysql://aa-model-db.c24oooc4uqmr.us-west-2.rds.amazonaws.com:3306/model_service?autoReconnect=true"
+  user: "aa_user"
```

Comment:
?? this is expedia-specific. are you just trying to get it to work? if so that's fine, but we need to remove it and not forget to do so